protopipe/mva/train_model.py

```diff
@@ -65,7 +65,7 @@ def split_data(
             weight = np.ones(len(self.data_train))
             weight_train = weight / sum(weight)
 
-        elif self.case in "classifier":
+        else:
```

Comment:
this may be error prone since you never check that self.case is one of ['regressor','classifier'].  If it is mistyped, you would always get a classifier.  So either check in `__init__` that it is one of the  expected values, or check explicitly here for "classifier" and have a else case for mistypes that raises an error.

Reply:
I moved this check directly in the main script and raised an error if the model used is not one of the supported ones (but technically we can support many more than before with the new code, but it needs to be tested)